Code/GraphMol/Kekulize.cpp

```diff
@@ -144,7 +144,7 @@ void markDbondCands(RWMol &mol, const INT_VECT &allAtms,
         riVect.begin(), riVect.end(),
         [&isRingNotCand](int ri) { return isRingNotCand.test(ri); });
     if (!at->getAtomicNum() && nonArNonDummyNbr < numAtomRings &&
-        numNonCandRings < numAtomRings) {
+        std::cmp_less(numNonCandRings, numAtomRings)) {
```

Comment:
it would be better to declare `numNonCandRings` to be `size_t` instead of using `auto`